kubemarine/core/defaults.py

```diff
@@ -545,3 +546,12 @@ def prepare_for_dump(inventory, copy=True):
 
     return dump_inventory
 
+
+def manage_true_false_values(inventory, cluster):
+    # Check undefined values for plugin.name.install
+    for plugin_name, plugin_item in inventory["plugins"].items():
+        # Check install value
+        if utils.true_or_false(plugin_item.get('install', False)) == 'undefined':
+            raise ValueError(f"Found unsupported value for plugin.{plugin_name}.install: {plugin_item['install']}")
```

Comment:
Found one more place `admission.label_namespace_pss`. And currently looks like we can just convert string to boolean here and revert all changes in `install` property usages.

Reply:
Was added bool concertation for plugin.install values